ext/Draw/CDraw3DLine.cpp

```diff
@@ -58,10 +56,6 @@ void CDraw3DLine::draw(
     auto buff = upStreamBuff->getBuffer();
     m_desc->mapVertexAttrBuffer(buff,EVAI_ATTR0,ECPA_THREE,ECT_FLOAT,sizeof(S3DLineVertex), offsetof(S3DLineVertex, Position[0]) + m_offsets[0]);
     m_desc->mapVertexAttrBuffer(buff,EVAI_ATTR1,ECPA_FOUR,ECT_FLOAT,sizeof(S3DLineVertex), offsetof(S3DLineVertex, Color[0]) + m_offsets[0]);
-    m_desc->mapIndexBuffer(buff);
-
-    m_meshBuffer->setIndexBufferOffset(m_offsets[1]);
-    m_meshBuffer->setIndexCount(2);
```

Comment:
index count still needs to be present... it tells how many vertices are to be drawn.